### PR TITLE
fix(coredump): remove unnecessary info

### PR DESCRIPTION
**Pull request by soyacz**
We don't need details about system modules in coredump details we post in issues. Removing it from the coredump event so it's never reposted there. Fixes also download instructions.

fixes: https://github.com/scylladb/scylla-cluster-tests/issues/7357

### Testing

- [ ]

### PR pre-checks (self review)

- [x] I added the relevant `backport` labels
- [x] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevant to this change (if needed)

### CODE REVIEW

File: sdcm/coredump.py [hunk 1]
```diff
@@ -211,7 +211,7 @@ def _upload_coredump(self, core_info: CoreDumpInfo):
                               "'%s' 'https://%s'" % (coredump, upload_url))
         download_url = 'https://storage.cloud.google.com/%s' % upload_url
         self.log.info("You can download it by %s (available for ScyllaDB employee)", download_url)
-        download_instructions = 'gsutil cp gs://%s .\ngunzip %s' % (upload_url, coredump)
+        download_instructions = 'gsutil cp gs://%s .\ngunzip %s' % (upload_url, coredump.rsplit('/', 1)[-1])
```
Comment (fruch) [inline sdcm/coredump.py:214]: Why not use pathlib ? i.e. it's a path we are handling here..

**Review by fruch — APPROVED** [review]
LGTM

**Review by michoecho — APPROVED** [review]
LGTM.

(Though I think that none of `coredumpctl info` is useful, neither the parts you filtered out nor the ones you left. I mean, it's not a bad idea to have it in the logs, but I wouldn't paste them into github issues. But that's just my opinion.)

**Review by vponomaryov — CHANGES_REQUESTED** [review]
Need to fix the unit tests

**Comment by fruch** [comment]
> LGTM.
> 
> (Though I think that none of `coredumpctl info` is useful, neither the parts you filtered out nor the ones you left. I mean, it's not a bad idea to have it in the logs, but I wouldn't paste them into github issues. But that's just my opinion.)

When you open a bug you can pick which information you think is important, I think the backtrace can be helpful, for example if you don't have the decoded backtraces from scylla.

Anyhow there's a link to Argus, and there you can find all of the data you need, and eventually people would stop cheery-picking the wrong data

**Comment by soyacz** [comment]
> Need to fix the unit tests

unit tests fixed

**Review by fruch — APPROVED** [review]
LGTM

**Pull request merged**